KaijuKiller/Assets/Scripts/SpawnManager.cs

```diff
@@ -7,11 +7,13 @@ public class SpawnManager : MonoBehaviour
     RoadSpawner roadSpawner;
```

Comment:
You do have a SpawnManager!

I would refactor this to contain the logic found in the spawners and simply spawn a lsit of game objects when SpawnTriggerEntered() is called